### PR TITLE
Improves the admin 'select equipment' verb

### PR DESCRIPTION
**Pull request by Kyep**
The "select equipment" verb is a button used by admins to dress up characters for events.
This PR tweaks it to fix some issues and add a few new options.

Major changes:
- All costumes now include ID cards. Including "as job..." options.
- ERT options now actually work. You can outfit a character as any kind of ert (amber/red/gamma) and any role in that ERT (sec/medical/eng/commander).
- "Spy", "NT VIP Guest", and "survivor" have been added to the list of choices. Spy is a cheap traitor with a silenced gun and emag. NT VIP Guest is a centcom guest with no gear. Survivor is a civilian who survived some terrible event.

Other Changes:
- reaper: starts out dust implanted
- nanotrasen officer: renamed to nt navy officer
- nanotrasen captain: renamed to nt navy captain
- special ops officer: renamed to nt special ops officer, with different (more combat-oriented) loadout than joining as this job from the job screen
- all NT CC outfits: changed headset to centcom, added dust/death implanters to backpacks
- Tunnel Clown: added full toolbelt

:cl: Kyep
tweak: Improved admin 'select equipment' verb. ERT option now works, costumes get IDs and proper headsets, two new costumes added, and some outfits minorly tweaked (e.g: tunnel clowns get toolbelts).
/:cl:

### CODE REVIEW

**Comment by CrAzYPiLoT-SS13** [comment]
While you're at it could you refactor the 'Do you want to strip the mob
before...' dialog? Currently, it's a shitty dialog where you choose 1 or 0
from the list, where a yes/no button dialog could work much better.

**Comment by Kyep** [comment]
@CrAzYPiLoT-SS13 Changed

**Comment by CrAzYPiLoT-SS13** [comment]
Thanks a ton.

**Comment by Fox-McCloud** [comment]
Rather than spam comments....

Please no `if (blah)`. Only `if(blah)`

Same with else, while, etc.

**Comment by Kyep** [comment]
@Fox-McCloud changed

**Comment by Necaladun** [comment]
While you're at it, reaper could use work on the secure briefcase. Spawning it locked already is odd.

**Comment by Kyep** [comment]
@Necaladun Originally, in an earlier version of this PR, I made it start out unlocked. Then I realized there's no point in having an unlocked briefcase, because that's just a backpack you can't put away.
Currently, the briefcase starts out locked. But, there is no code set. First time you use it, you set the code. Then you unlock it with that code. From then on, you can unlock/relock it at will. This helps you stash items should Security want to search you. It also looks pretty stylish.

I could just do away with it entirely. Give them a storage implant or something instead. But at the moment, I've left them with the briefcase. They're the only character to get one, so its kinda a unique thing.

File: code/modules/admin/verbs/debug.dm [hunk 1]
```diff
@@ -595,36 +595,41 @@ But you can call procs that are of type /mob/living/carbon/human/proc/ for that
...
 		)
-	var/dostrip = input("Do you want to strip [M] before equipping them? (0=no, 1=yes)", "STRIPTEASE") as null|anything in list(0,1)
-	if(isnull(dostrip))
-		return
+	var/dostrip = 0
+	switch(alert("Strip [M] before dressing?", "Strip?", "Yes", "No", "Cancel"))
+		if("Yes")
+			dostrip = 1
+		if("Cancel")
+			return
```
Comment (TheDZD) [inline code/modules/admin/verbs/debug.dm]: You broke the alert, hitting "No" isn't going to do anything.

Comment (TheDZD) [inline code/modules/admin/verbs/debug.dm]: Nevermind, tested this and it works.

**Pull request merged**